app/src/main/java/io/apicurio/registry/storage/impl/SimpleMapRegistryStorage.java

```diff
@@ -5,6 +5,7 @@
 import java.util.Set;
 import java.util.concurrent.ConcurrentHashMap;
 
+import org.apache.kafka.connect.errors.AlreadyExistsException;
```

Comment:
this seems wrong

Reply:
It is wrong.  Good catch.  Fortunately this class is being deleted. :)